### PR TITLE
Refactored colors into the `theme`

### PR DESCRIPTION
**Pull request by djoyal**
`Grommet` allows us to add custom-name colors to the `theme` so that we can easily use them inside of `Grommet`'s components.

This allows us to do stuff like `<Button color="my_color" />` instead of having to import our colors everywhere.

I updated the references to the COLORS object in the component/loginModal files.

I also put the `FACEBOOK_BLUE` color into a local variable for the `LoginForm`, since it doesn't really belong in our global theme.

### CODE REVIEW

File: packages/web/lib/theme.tsx [hunk 1]
```diff
@@ -34,8 +34,8 @@ const inputField = {
 const theme = {
   global: {
     colors: {
-      brand: COLORS.BRAND,
-      focus: COLORS.BRAND,
+      ...BRAND_COLORS,
```
Comment (djoyal) [inline packages/web/lib/theme.tsx:37]: Note the use of the spread operator (`...`) here to inject `BRAND_COLORS` into the them. If you want to add more colours, you should add them to `BRAND_COLORS`.

Comment (jfplataroti) [reply packages/web/lib/theme.tsx:37]: nicee!!

File: packages/web/components/loginModal/loginForm.tsx [hunk 2]
```diff
@@ -52,7 +51,7 @@ export function LoginForm() {
         fill="horizontal"
         margin={{ vertical: "4px" }}
         label={t("sign_in_with_google")}
-        color={COLORS.GRAY_3}
+        color="gray_3"
```
Comment (jfplataroti) [inline packages/web/components/loginModal/loginForm.tsx:54]: nice, much better!

File: packages/web/components/loginModal/loginForm.tsx [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
 import { Box, Button, Heading, Paragraph } from "grommet";
 import { Apple, Facebook, Google } from "grommet-icons";
 import React from "react";
-
 import { useTranslation } from "../../i18n";
-import { useAuth0 } from "@auth0/auth0-react";
 import { AuthConnections } from "../../lib/auth";
 
-import { COLORS } from "../../lib/theme";
+const FACEBOOK_BLUE = "#468AF7";
```
Comment (jfplataroti) [inline packages/web/components/loginModal/loginForm.tsx:8]: makes sense to keep it only here 👍

**Review by jfplataroti — APPROVED** [review]
Nice! thanks for adding them to the theme! I really like this approach of being able to centralize everything in the UI library. :)

**Pull request merged**